tools/uswds-component-library-figma.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: default
+title: USWDS Component Library in Figma
+nav: Tools
+permalink: /tools/uswds-component-library-figma
+sidenav: tools-side-nav
+---
+
+# {{ page.title }}
+
+The [USWDS component library](https://www.figma.com/community/file/836611771720754351) is available and free to use through the Figma Community. Created by the [Truss.works team](https://truss.works/), it's a great place to start incorporating components into your designs.
+
+## Getting started
+
+### Duplicating the USWDS component library
+
+To view and duplicate files from the Figma Community, login or create a [Figma account](https://www.figma.com/).
+
+1. Navigate to the community tab in the left-hand side navigation
+1. Search for "USWDS" in the search bar
+1. Open the USWDS file by Truss
+1. Duplicate the file
+
+![GIF demonstrating how to navigate through Figma's community page to duplicate the U.S. Web Design System library created by Truss.]({{ site.url }}/{{site.baseurl}}/assets/img/uswds-component-library-getting-started.gif)
```

Comment:
Love the gifs, but one thing we should go back and check is how they might affect page performance once it's live.

Reply:
Actually, I just realized these are multiple megabytes. Anyway we can trim these down?